src/DIRAC/Core/scripts/dirac_cert_convert.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+########################################################################
+# File :   dirac-cert-convert
+# Author : Andrii
+########################################################################
+"""
+Script converts the user certificate in the p12 format into a standard .globus usercert.pem and userkey.pem files.
+Creates the necessary directory, $HOME/.globus, if needed. Backs-up old pem files if any are found.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+__RCSID__ = "$Id$"
+
+import os
+import sys
+import shutil
+from getpass import getpass
+from datetime import datetime
+
+from DIRAC import gLogger
+from DIRAC.Core.Base import Script
+from DIRAC.Core.Utilities.Subprocess import shellCall
+from DIRAC.Core.Utilities.DIRACScript import DIRACScript
+
+
+@DIRACScript()
+def main():
+  Script.parseCommandLine(ignoreErrors=True)
+
+  positionalArgs = Script.getPositionalArgs()
+  if len(positionalArgs) == 0:
+    gLogger.fatal("User Certificate P12 file is not given.")
+    sys.exit(1)
+
+  p12 = positionalArgs[0]
+  if not os.path.isfile(p12):
+    gLogger.fatal("%s does not exist." % p12)
+    sys.exit(1)
+
+  globus = os.path.join(os.environ["HOME"], '.globus')
+  if not os.path.isdir(globus):
+    gLogger.notice("Creating globus directory")
+    os.mkdir(globus)
+
+  p12Path = os.path.join(globus, os.path.basename(p12))
+  cert = os.path.join(globus, 'usercert.pem')
+  key = os.path.join(globus, 'userkey.pem')
+
+  nowPrefix = '.' + datetime.now().isoformat()
+  for old in [p12Path, cert, key]:
+    if os.path.isfile(old):
+      gLogger.notice("Back up %s file." % old)
+      shutil.copy(old, old + nowPrefix)
+
+  shutil.copy(p12, p12Path)
+
+  password = getpass('Enter password for p12:')
+  password = ' -password pass:%s' % password
+
+  # new OpenSSL version require OPENSSL_CONF to point to some accessible location',
+  result = shellCall(900, "export OPENSSL_CONF=/tmp")
```

Comment:
Does `shellCall` create a  "persistent" environment where this is still set in the next shellCalls to openssl?

Reply:
fixed